lib/server/server.js

```diff
@@ -14,10 +14,9 @@
 
 import OktaAuthBase from '../OktaAuthBase';
 import fetchRequest from '../fetch/fetchRequest';
-import { postToTransaction } from '../tx';
 import { getUserAgent } from '../builderUtil';
 import serverStorage from './serverStorage';
-import PACKAGE_JSON from '../../package.json';
+const PACKAGE_JSON = require('../../package.json');
```

Comment:
`import * as PACKAGE_JSON  from '../../package.json'` may work

Reply:
In this case, the issue is that importing a JSON file via typescript requires the `esModuleInterop` flag to be turned on. I specifically want this flag off because if we have it on the requirement of using this flag is passed along to other users of our ES module, such as `okta-angular`.

Reply:
So the same issue for `node-cache`?

Reply:
Yes, `node-cache` is a `commonJS` module. It can only be imported by using the `allowSyntheticDefaultImports` and `esModuleInterop` flags.